.circleci/config.yml

```diff
@@ -70,14 +70,6 @@ commands:
           command: |
               sudo ln -s /lib/x86_64-linux-gnu/libtinfo.so.6 /lib/x86_64-linux-gnu/libtinfo.so.5
 
-  install-xcpretty:
```

Comment:
I guess this conflicts with the doc generation somehow?

Reply:
Ah, should have left a comment.
Turns out xcpretty is installed by default on the container on CircleCI and ... the one we installed wasn't used because it landed in a different path anyway.